Alex-App/src/app/my-service.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from "@angular/core";
+import { HttpClient, HttpHeaders } from "@angular/common/http";
+import { NewMember } from "./models/newMember.model";
+
+@Injectable({
+  providedIn: "root"
+})
+export class MyServiceService {
+  baseUrl: string = `https://130132ee-94fa-4470-ae76-062094b6ec43.mock.pstmn.io/api/Members/Email`;
+
+  postUrl: string = `https://130132ee-94fa-4470-ae76-062094b6ec43.mock.pstmn.io/api/Members/Emails`;
+
+  httpOptions = {
+    headers: new HttpHeaders({
+      "Content-Type": "application/json",
+      "x-api-key": "3bad84c8a9e8450ea2e2fe160bc6dab4"
+    })
+  };
+
+  constructor(private http: HttpClient) {}
+
+  getMembers(): Promise<any> {
```

Comment:
You can put a type instead of `any`. e.g. `Member[]` or whatever it is

Reply:
So, when I go to change this part of the code i get an error in the list component that says 'data' does not exist on type 'Member[]' changing it to any causes the error to go away, and data exists for sure, because it's what's being printed. I fee like im missing something with the Member interface maybe?